fvtr/systemtap/systemtap.exp

```diff
@@ -49,17 +49,21 @@ if { $env(AT_CROSS_BUILD) == "yes" } {
 	set install_prefix $env(AT_DEST)
 }
 
+# Starting with glibc 2.34 (AT 15.0) libpthread was moved into libc.so
 if { $TARGET32 } {
-	set lib [glob $install_prefix/lib/libpthread-*.so]
-	if {[check_systemtap $readelf_exe $lib]} {
-		set ret 1
-	}
+	set libpath "$install_prefix/lib"
+} else {
+	set libpath "$install_prefix/lib64"
 }
-if { $TARGET64 } {
-	set lib [glob $install_prefix/lib64/libpthread-*.so]
-	if {[check_systemtap $readelf_exe $lib]} {
-		set ret 1
-	}
+
+if { $::env(AT_MAJOR_VERSION) < 15.0 } {
```

Comment:
Can we test for `AT_GLIBC_VER < 2.34` instead?
That works even if someone decides to build an AT 14.0 with glibc 2.34.

Reply:
Sure. Changed in the latest push.